packages/frontend/src/components/NavBar/SettingsMenu/index.tsx

```diff
@@ -50,11 +48,7 @@ const SettingsMenu: FC = () => {
                             role="menuitem"
                             icon={<IconDatabase size={17} />}
                             text="Project settings"
-                            onClick={() => {
-                                history.push(
-                                    `/generalSettings/projectManagement/${activeProjectUuid}`,
-                                );
-                            }}
+                            href={`/generalSettings/projectManagement/${activeProjectUuid}`}
```

Comment:
We should use `<LinkMenuItem />` so we can have the best of both worlds: allow open in the tab and avoid page reload with left click